src/myna/database/peregrine.py

```diff
@@ -171,6 +171,15 @@ def load(self, metadata_type, part=None, layer=None):
                 df.write_parquet(file_database, compression="lz4")
             return file_database
 
+        elif metadata_type == metadata.MeltOrder:
+            # Assume that the print order is the same as the part ID order if there is
```

Comment:
Updated the comment to be more clear. We need this property for the `build_region` type to avoid the assumption that all parts print at the same time, but it is not a piece of metadata I currently know how to extract from the Peregrine metadata.